e2e/test-1.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+
+test('Remove item from cart', async ({ page }) => {
+  await page.goto('https://localhost:19888/');
```

Comment:
As per @jamesmontemagno as "we don't have eShop deployed with base URL. So, as of now Just remove the current Github action until we find a better solution and folks can run them locally"
Resolving this comment

Reply:
The baseURL is nothing to do with deployment although it can be also used in the webserver.  It’s about not having to duplicate code as each time you put page.goto localhost 1888 by using baseURL it means you can use page.goto with just a slash and it will always go to the url defined in the config. Then think ahead. You have a thousand tests with this repeated code. Thats a lot of refactoring if you change the url which will eventually happen. By using baseURL in the config you will only have to change it once and all your tests still work. Hope that makes sense